src/formio/utils.js

```diff
@@ -14,4 +14,38 @@ const escapeHtml = source => {
   return pre.innerHTML.replace(/"/g, '&quot;').replace(/'/g, '&apos;').replace(/&/g, '&amp;');
 };
 
-export {applyPrefix, escapeHtml};
+const setErrorAttributes = (elements, hasErrors, hasMessages, parentElement) => {
+  // Update the attributes 'aria-invalid' and 'aria-describedby' using hasErrors
+  elements.forEach(element => {
+    const errorMessageElementId = parentElement.querySelector('[ref="messageContainer"]')?.id;
```

Comment:
Instead of passing `parentElement`, I think it makes more sense to pass the `this` object (which is the component) instance and then you can use `component.refs.messageContainer` - that's how Formio's ref mechanism is supposed to be used.

Reply:
I've changed it to:

```
setErrorAttributes(..., ... ,..., this.refs.messageContainer.id);
```

Seeing how this function only needs the id of the message container